aka.js

```diff
@@ -324,6 +350,155 @@ function spawn_update_check(update_file_path) {
   }).unref();
 }
 
+// Get random tips or update available statement (if applicable)
+function get_epilogue() {
+  const update_available = module.exports.update_available ? (Object.keys(module.exports.update_available).length !== 0) : false;
+  if (update_available) {
+    return module.exports.terminal.update_statement(module.exports.update_available.current, module.exports.update_available.latest);
+  } else {
+    return module.exports.random_tips[Math.floor(module.exports.random_tips.length * Math.random())]
+  }
+}
+
+// Print UI and any error messages
+function print_ui(cliui, errorMessage) {
+  if (errorMessage) {
+    cliui.span(`\n${errorMessage}`);
+  } else {
+    cliui.span(`\n${get_epilogue()}`);
+  }
+  console.log(cliui.toString());
+}
+
+// Print help for a specific command group
+function print_group_help(argv, group) {
+  // Initialize UI
+  let errorMessage = ''
+  const ui = require('cliui')({width: process.stdout.columns})
+  ui.div(chalk.bold('\nAkkeris CLI Help\n'))
```

Comment:
Can we use the `terminal.markdown` instead of introducing new dependencies (e.g., chalk) on the CLI? We need to be careful adding dependencies, it's easy to do a `npm install` but if the dependent license, or its dependency licenses are incompatible with Apache-2 we get in a bit of quandary. In addition this functionality already exists, so using chalk adds an additional 18 packages (according to npmjs). If you need help using the markdown let me know.